lib/src/test/java/com/auth0/jwt/JWTVerifierTest.java

```diff
@@ -410,9 +421,25 @@ public void shouldThrowOnNegativeCustomLeeway() throws Exception {
 
     // Expires At
     @Test
-    public void shouldValidateExpiresAtWithLeeway() throws Exception {
+    public void shouldValidateExpiresDateAtWithLeeway() throws Exception {
+        Clock clock = mock(Clock.class);
+        when(clock.getNow()).thenReturn(Instant.ofEpochMilli(DATE_TOKEN_MS_VALUE + 1000));
+
+        String token = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE0Nzc1OTJ9.isvT0Pqx0yjnZk53mUFSeYFJLDs-Ls9IsNAm86gIdZo";
+        JWTVerifier.BaseVerification verification = (JWTVerifier.BaseVerification) JWTVerifier.init(Algorithm.HMAC256("secret"))
+                .acceptExpiresAt(2);
+        DecodedJWT jwt = verification
+                .build(clock)
+                .verify(token);
+
+        assertThat(jwt, is(notNullValue()));
+    }
+
+
+    @Test
+    public void shouldValidateExpiresInstantAtWithLeeway() throws Exception {
         Clock clock = mock(Clock.class);
-        when(clock.getToday()).thenReturn(new Date(DATE_TOKEN_MS_VALUE + 1000));
+        when(clock.getNow()).thenReturn(Instant.ofEpochMilli(DATE_TOKEN_MS_VALUE + 1000));
```

Comment:
Again, if this tests legacy behavior it might be good to keep everything as is. You have several changes like this one on this file.

Reply:
See above response for why we need to mock `getNow` instead of `getToday`.